website/addons/base/views.py

```diff
@@ -472,6 +494,16 @@ def addon_deleted_file(auth, node, **kwargs):
     """
     # Allow file_node to be passed in so other views can delegate to this one
     trashed = kwargs.get('file_node') or TrashedFileNode.load(kwargs.get('trashed_id'))
+    if kwargs.get('path'):
+        file_path = kwargs.get('path')
+    else:
+        file_path = trashed.path
+    if trashed.name:
+        file_name = trashed.name
+    else:
+        file_name = os.path.basename(file_path)
```

Comment:
These if else statements can be made a little less verbose with some ```.get()``` and some ```getattr()``` use with defaults! Something like:

```python
file_path = kwargs.get('path', trashed.path)
file_name = getattr(trashed, 'name', os.path.basename(file_path))
```